data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,2403 @@
+					goto soldier
+			`He does not answer your question directly, "If you really most know, these supplies were used to prevent an outbreak or a disease on soldiers, that could have affected all of us. But if the soldiers are not fit to protect us, then civilians will suffer. The Hai would not dare to kill us of course, but they would destroy our way of life, our spirits."`
+			
+			label soldier
+			`	"Do I look like a soldier to you? Does this medic look like one? No, we Unfettered only dare to think in another way, in truth quite a few of us would like to be artists and it is a shame we have to fight for our right to live another way than the one the Hai allow, to break of the prison they have casted next to our systems!"`
+
+
+
+mission "Unfettered: Prove Your Strength"
+	name `Defeat the captain`
+	description `A young Unfettered captain has provoked you in a duel, disable his ship.`
+	source "Darkcloak"
+	to offer
+		has "Unfettered Jump Drive 2: offered"
+		has "Unfettered: Bring Medical Supplies: done"
+		has "armament deterrence" > 5
+	on offer
+		conversation
+			`You start wandering in the spaceport, and notice that everything around here is so... simple, efficient, ordered. There seems to be more warriors here than on the other Unfettered planets, or at least they are less affrayed to wonder around with their weapon, they seem ready for action at any time, you are happy not to be their enemy, or at least, for now.	As you continue walking you notice someone speaking to a crowd of people. They seem very familiar with each others, and it seems they have noticed you too, the one who was adressing the others approaches you.`
+			`	"This is the monkey you speak so highly off? The one who fears not to come into our space, to fight our ships?" He asks the others, apparently mocking you. This could get you into trouble.`
+			choice
+				`	"I fear nothing. Do you?"`
+					goto provoke
+				`	(Say nothing.)`
+				`	(Run away.)`
+					decline
+			`	"See? The monkey dares not speak! He would not stand a chance in combat, our modest ship would bring him straight out of the sky, but we should not even try or we could hurt it. But my friends, there is no shame for him to have... With a ship looking like that one I would be hesitant to fight too!"	He seems very full of himself, maybe you should teach him a lesson.`
+			choice
+				`	"How dare you call the <ship>?"`
+				`	"I have no need to prove myself to a rodent."`
+				`	(Go, he is not worth it.)`
+					decline
+			
+			label provoke
+			`	"So the monkey can speak, let us see if it can fight as well!"	He seems very full of himself, time for his lesson. Or perhaps you are the one who needs to learn humility... After a cheering from his crowd, he speaks to you directly this time. "You and me, in battle, leave your fleet parked. Meet me in the sky when you are ready, except if you lack the courage to fight me. This is not to be a fight to the death, monkey."`
```

Comment:
I don't know much about ES style norms, but yes it is indeed directed to the player, because the captain would not say something like that.
Maybe I should add something like 'you think to yourself'
Or if this doesn't fit the norms make it be a dialog if you fail the mission that you are the one in need of humility?

Reply:
The fail dialogue would probably be more appropriate for the direct message, but 'think to yourself' is good enough way to leave it in this section as well.

Reply:
yeah there is no chance he defeats you actually because the mission only trigger if you have enough strength to beat him
Even if he does disable you I have no way to check for that, all that can be known is if you destroy him